scripts/IdentifyOpportunity.py

```diff
@@ -0,0 +1,112 @@
+# This is an application similar screener and it only tries to provide you with some good insight by looking and comparing different trading pairs in an exchange
+# By default the format_status command refreshes at .1 seconds, so I recommend changing it to 10 or bigger number for this script to run without any issues.
+# You can go to the line 144 in status_command.py file and update the number to 10
+from hummingbot.strategy.script_strategy_base import ScriptStrategyBase
+import pandas as pd
+from binance.client import Client
+from hummingbot.core.data_type.common import PriceType
+
+class IdentifyOpportunity(ScriptStrategyBase):
+    """
+    This example shows how to add a custom format_status to a strategy and query the order book.
+    Run the command status --live, once the strategy starts.
+    """
+
+    exchange = "binance_us_paper_trade"
+    top_nrows=int(5)
+	
+	#Below are binance_us api_key and secrets I used to get all the assets from the exchange
+    api_key = <api_key>
+    api_secret = <api_secret>
+
+
+
+    client = Client(api_key, api_secret, tld='us')
+
+    #Below section of the code gives us all the assets that are tradable
+    columns = ['symbol','baseAsset','quoteAsset','permissions','ocoAllowed','isMarginTradingAllowed','isSpotTradingAllowed']
+    tradable_assets=[]
+    for i in client.get_exchange_info()['symbols']:
+      if (i['status']=='TRADING'):
+        #print(list(map(i.get,columns)))
+        tradable_assets.append(list(map(i.get,columns)))
+    tradable_assets = pd.DataFrame(tradable_assets,columns=columns)
+    tradable_assets_usd = tradable_assets[tradable_assets['quoteAsset']=='USD']
+    # display(tradable_assets_usd)
+    # print("Number of tickers that can be bought with Margin - {}".format(len(tradable_assets_usd[tradable_assets_usd['isMarginTradingAllowed']==True])))
+         
+        
+    markets = {
```

Comment:
I see that you are getting all the assets, but then you are only initializing 3 of them.

Reply:
yeah, I think I pushed the test script. If you can comment line 41 and uncomment line 40 the script should work.

I also recorded a loom video just now.

https://www.loom.com/share/1873c71ca18f407288b8b531b380749b